spec/models/course/assessment_spec.rb

```diff
@@ -144,10 +144,8 @@
     context 'after assessment was initialized' do
       subject { build(:assessment) }
 
-      it 'builds a folder' do
-        expect(subject.folder).to be_present
-
-        expect(subject).to be_valid
+      it 'sets the folder to have the same attributes as the assessment' do
+        subject.save
```

Comment:
Yeah, I think that's why the description was `'builds a folder'` before, and the subsequent expectations are after validations. 

How about just ` subject { create(:assessment) }` and change the context description accordingly ?